### PR TITLE
Luxury pens

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Someone brought up the idea of having custom pens for specific jobs that i thought sounded cool, and then Emisse said they wanted special pens for lawyers librarians and reporters specifically so i did that. In the future hopefully these pens can write with a fancy font different to normal pens, like Merriweather for example.

image

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image
image

**Changelog**

:cl: Ubaser
- add: Lawyers, librarians and reporters now spawn with a luxury pen in their PDA.

### CODE REVIEW

**Comment by UbaserB** [comment]
i just realised i need to undraft it if i want to check what is erroring

**Comment by deltanedas** [comment]
pda component duped

just check console for errors

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml [hunk 1]
```diff
@@ -250,6 +250,22 @@
...
+  id: LuxuryPen
+  description: A fancy and expensive pen that you only deserve to own if you're qualified to handle vast amounts of paperwork.
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/bureaucracy.rsi
+    state: luxury_pen
+  - type: Item
+    sprite: Objects/Misc/bureaucracy.rsi
+    heldPrefix: luxury_pen
+    size: Tiny
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Misc/paper.yml:267]: 262, 265 and 267 lines can be removed since inherited

**Comment by JoeHammad1844** [comment]
on TG these pens can do the 'you feel a little prick!' thing hypos do, would be fun to add that, aswell

**Pull request merged**